test/parallel/test-tls-getcipher.js

```diff
@@ -25,6 +25,7 @@ server.listen(common.PORT, '127.0.0.1', function() {
   var client = tls.connect({
     host: '127.0.0.1',
     port: common.PORT,
+    ciphers: cipher_list.join(':'),
```

Comment:
Described here: https://github.com/iojs/io.js/pull/826#issuecomment-74374449

I went with the first option.